src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @InjectMocks
+    private MenuGroupService menuGroupService;
+
+    @Test
+    @DisplayName("메뉴 그륩을 등록한다")
+    void createMenuGroup() {
+        MenuGroup 피자 = new MenuGroup(1L, "피자");
+        when(menuGroupDao.save(any())).thenReturn(피자);
```

Comment:
```suggestion
        given(menuGroupDao.save(any())).willReturn(피자);
```
BDDMockito를 사용하면 given when then 스타일로 작성할수 있어요 😀

Reply:
엇 STEP2에서는 given 으로 한번 진행해보겠습니다~!~!